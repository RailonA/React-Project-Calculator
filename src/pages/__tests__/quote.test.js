import React from 'react';
import { render } from '@testing-library/react';
import Quote from '../quote';

describe('Quote page', () => {
  it('should match the snapshot', () => {
    const { asFragment } = render(<Quote />);
    expect(asFragment(<Quote />)).toMatchSnapshot();
  });
});
