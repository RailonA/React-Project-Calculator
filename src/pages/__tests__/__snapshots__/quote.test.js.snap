// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote page should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="qouteDiv"
  >
    <div>
      <p>
        Pure mathematics is, in its way, the poetry of logical ideas.
      </p>
      <p
        class="qouteDiv-p"
      >
        â€” Albert Einstein, German theoretical physicist
      </p>
    </div>
  </div>
</DocumentFragment>
`;
